src/Blog/Entity/Comment.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Blog\Entity;
+
+use App\Entity\User;
+use Cycle\Annotated\Annotation\Column;
+use Cycle\Annotated\Annotation\Entity;
+use Cycle\Annotated\Annotation\Relation\BelongsTo;
+use Cycle\Annotated\Annotation\Table;
+use Cycle\Annotated\Annotation\Table\Index;
+use DateTimeImmutable;
+
+/**
+ * @Entity(
+ *     mapper="App\Blog\Comment\CommentMapper",
+ *     constrain="App\Blog\Comment\Scope\PublicScope"
+ * )
+ * @Table(
+ *     indexes={
+ *         @Index(columns={"public","published_at"})
+ *     }
+ * )
+ */
+class Comment
+{
+    /**
+     * @Column(type="primary")
+     */
+    private ?int $id = null;
+
+    /**
+     * @Column(type="bool", default="false")
+     */
+    private bool $public = false;
+
+    /**
+     * @Column(type="text")
+     */
+    private string $content;
+
+    /**
+     * @Column(type="datetime")
+     */
+    private DateTimeImmutable $created_at;
+
+    /**
+     * @Column(type="datetime")
+     */
+    private DateTimeImmutable $updated_at;
+
+    /**
+     * @Column(type="datetime", nullable=true)
+     */
+    private ?DateTimeImmutable $published_at = null;
+
+    /**
+     * @Column(type="datetime", nullable=true)
+     */
+    private ?DateTimeImmutable $deleted_at = null;
+
+    /**
+     * @BelongsTo(target="App\Entity\User", nullable=false, load="eager")
+     * @var User|\Cycle\ORM\Promise\Reference
+     */
+    private $user = null;
+    private ?int $user_id = null;
```

Comment:
Do we need to map it if it's not used anywhere?

Reply:
This is field that automatically generated by cycle
This field can be used in binding process and may be mentioned in SQL queries
Right now we have not use cases and this field is for reference only and not for functional.